src/picongpu/include/plugins/output/GatherSlice.hpp

```diff
@@ -188,6 +188,7 @@ struct GatherSlice
                 insertData(dstBox, srcBox, head->node.offset, head->node.maxSize);
             }
 
+            __deleteArray(fullData);
```

Comment:
`__deleteArray()` is a PMacc macro which sets the pointer after it is deleted to `NULL`. This macro is newer than tthis class, therefore it is currently not used in the destructor or at other parts of this class.